pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1175,7 +1175,11 @@ impl RemoteTimelineClient {
                     // at info level at first, and only WARN if the operation fails repeatedly.
                     //
                     // (See similar logic for downloads in `download::download_retry`)
-                    if retries < FAILED_UPLOAD_WARN_THRESHOLD {
+
+                    let is_simulated = cfg!(feature = "testing")
+                        && e.root_cause().is::<remote_storage::SimulatedError>();
+
+                    if retries < FAILED_UPLOAD_WARN_THRESHOLD || is_simulated {
```

Comment:
I guess the `is_simulated` condition is intended to avoid the need for allowing the log line in the tests?

Reply:
Yes, I just squashed the commits from #5217 to this, please ignore. I needed it for the test case debugging which turned out to be silly failure on my part: [b0b3027](https://github.com/neondatabase/neon/pull/5198/commits/b0b3027b6057bdbdf5a3970678dee7f40a306604)

Reply:
I've now dropped this.